.eleventy.js

```diff
@@ -1,8 +1,35 @@
 /* eslint-disable @typescript-eslint/no-var-requires, import/no-dynamic-require, global-require  */
+const { parse } = require('@babel/parser')
+const { transform } = require('@svgr/core')
+
+require('@babel/register')({
+  extensions: ['.svg'],
+  babelrc: false,
+  configFile: false,
+  presets: [
+    ['@babel/preset-env'],
+    ['@babel/preset-react', {
+      runtime: 'automatic',
+    }],
+  ],
+  plugins: [{
+    parserOverride(code, opts) {
+      return parse(
+        transform.sync(code, { plugins: ['@svgr/plugin-jsx'], jsxRuntime: 'automatic' }, {
+          componentName: 'SvgComponent',
+        }),
+        opts,
+      )
+    },
+  }],
+})
+
```

Comment:
When with svgr because I can isolate the SVG file, Otherwise if I went with babel inline, I would need to do the transformation on the TSX via babel.

Reply:
switched this to svg inline